src/main/java/fi/tarina/tarinamittaus/Configs/PersistenceJPAConfigAwsDev.java

```diff
@@ -1,147 +1,147 @@
+//         String endpoint = env.getProperty("spring.aws.secretsmanager.endpoint");
+//         String region = env.getProperty("spring.aws.secretsmanager.region");
+
+//         AwsClientBuilder.EndpointConfiguration config = new AwsClientBuilder.EndpointConfiguration(endpoint, region);
+//         AWSSecretsManagerClientBuilder clientBuilder = AWSSecretsManagerClientBuilder.standard();
+//         clientBuilder.setEndpointConfiguration(config);
+//         AWSSecretsManager client = clientBuilder
+//                 .withCredentials(new EC2ContainerCredentialsProviderWrapper())
+//                 .build();
+
+
+//         ObjectMapper objectMapper = new ObjectMapper();
+//         JsonNode secretsJson = null;
+
+//         ByteBuffer binarySecretData;
+//         GetSecretValueRequest getSecretValueRequest = new GetSecretValueRequest()
+//                 .withSecretId(secretName);
+//         GetSecretValueResult getSecretValueResponse = null;
+//         try {
+//             //täsä tulee eroro
+//             getSecretValueResponse = client.getSecretValue(getSecretValueRequest);
+
+//         } catch (ResourceNotFoundException e) {
+//             logger.error("The requested secret " + secretName + " was not found);");
+//         } catch (InvalidRequestException e) {
+//             logger.error("The request was invalid due to: " + e.getMessage());
+//         } catch (InvalidParameterException e) {
+//             logger.error("The request had invalid params: " + e.getMessage());
+//         }
+
+//         if (getSecretValueResponse == null) {
+//             return null;
+//         }
+
+//         // Decrypted secret using the associated KMS CMK
+//         // Depending on whether the secret was a string or binary, one of these fields will be populated
+//         String secret = getSecretValueResponse.getSecretString();
+//         if (secret == null) {
+//             logger.error("The Secret String returned is null");
+//             return null;
+//         }
+//         try {
+//             secretsJson = objectMapper.readTree(secret);
+//         } catch (IOException e) {
+//             logger.error("Exception while retreiving secret values: " + e.getMessage());
+//         }
+
+
+//         logger.error("Secrets json - " + secretsJson);
+//         String host = secretsJson.get("host").textValue();
+//         String port = secretsJson.get("port").textValue();
+//         String dbname = secretsJson.get("dbname").textValue();
+//         String username = secretsJson.get("username").textValue();
+//         String password = secretsJson.get("password").textValue();
+//         String schema = secretsJson.get("schema_tarina").textValue();
+//         dataSource.setUrl("jdbc:postgresql://" + host + ":" + port + "/" + dbname);
+//         dataSource.setUsername(username);
+//         dataSource.setPassword(password);
+//         dataSource.setSchema(schema);
+
+//         return dataSource;
```

Comment:
Pitäskö nää kaikki kommentoidut blokit vaa poistaa? Samoin noi tuolla alempana?

Reply:
Joo kyllähän ne voisi. Palaan tähän lähipäivinä.

Reply:
Joo poistin nyt noi configit. Näitä swagger-annotaatiota ym. voisi kattoa myöhemmin. Ainakin readmestä pitää kuitenkin tehdä jatkotiketti niin siinä samalla voisi kattoa näitä tarkemmin.